2-mandatory/1-create-functions.js

```diff
@@ -44,7 +50,13 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(numbers) {
+  return numbers.map((number) => {
+    if (number > 100) {
```

Comment:
The instruction is saying if number >= 100 then return "100%". Please amend your if statement